src/Rules/UnusedPublicClassMethodRule.php

```diff
@@ -4,7 +4,6 @@
 
 namespace TomasVotruba\UnusedPublic\Rules;
 
-use Nette\Utils\Arrays;
```

Comment:
before this PR a class was used which is not declared in composer.json dependencies